src/UnitySampleProject/Assets/Samples/web3.unity SDK/2.5.0/Web3.Unity Samples/Scripts/Utilities/DisableGameObjectIfServiceNotActive.cs

```diff
@@ -7,32 +7,37 @@
 
 public enum ServiceType
 {
-    #if RAMP_AVAILABLE
-    Ramp,
-    #endif
-    Gelato,
-    Multicall
+    Ramp = 0,
+    Gelato = 1,
+    Multicall = 2
 }
 
 public class DisableGameObjectIfServiceNotActive : MonoBehaviour
 {
     [SerializeField] private ServiceType serviceType;
     private readonly Dictionary<ServiceType, Type> _typesDictionary = new()
     {
-        #if RAMP_AVAILABLE
+#if RAMP_AVAILABLE
         {ServiceType.Ramp, typeof(ChainSafe.Gaming.Exchangers.Ramp.IRampExchanger)},
-        #endif
+#endif
         {ServiceType.Gelato, typeof(IGelato)},
         {ServiceType.Multicall, typeof(IMultiCall)}
     };
 
     private void Awake()
     {
-        if (_typesDictionary.TryGetValue(serviceType, out var value))
-        {
-            gameObject.SetActive(Web3Accessor.Web3.ServiceProvider.GetService(value) != null);
-        }
+        if (serviceType != ServiceType.Ramp)
+            gameObject.SetActive(Web3Accessor.Web3.ServiceProvider.GetService(_typesDictionary[serviceType]) != null);
```

Comment:
this can instead be `gameObject.SetActive(_typesDictionary.TryGetValue(serviceType, out var value) && Web3Accessor.Web3.ServiceProvider.GetService(value) != null)`

Reply:
Great suggestion!